region/maridia/inner-pink.json

```diff
@@ -6222,12 +6235,86 @@
                   "note": "Doesn't require anything, but getting out does."
                 }
               ]
+            },
+            {
+              "id": 4,
+              "strats": [
+                {
+                  "name": "G-Mode Morph",
+                  "notable": false,
+                  "requires": [
+                    {"comeInWithGMode": {
+                      "fromNodes": [1],
+                      "mode": "any",
+                      "artificialMorph": true
+                    }},
+                    "Gravity",
+                    "h_canArtificialMorphJumpIntoIBJ"
```

Comment:
Getting to 4 looks like spike hits or diagonal IBJs

Reply:
It's actually quite easy with spring ball and I think it could be assumed for this difficulty. I'll add the diagonals and a note

Reply:
JK, the platform before this is the main problem. I could add an HBJ to pass those spikes, then a spring ball into IBJ ..